lib/webauthn/attestation_statement.rb

```diff
@@ -13,8 +16,11 @@ def self.from(format, statement)
       when ATTESTATION_FORMAT_FIDO_U2F
         require "webauthn/attestation_statement/fido_u2f"
         WebAuthn::AttestationStatement::FidoU2f.new(statement)
+      when ATTESTATION_FORMAT_PACKED
+        require "webauthn/attestation_statement/packed"
+        WebAuthn::AttestationStatement::Packed.new(statement)
       else
-        raise "Unsupported attestation format '#{attestation_format}'"
+        raise WebAuthn::AttestationStatement::Base::NotSupportedError, "Unsupported attestation format '#{format}'"
```

Comment:
Good catch about the undefined variable, thanks.

In an attempt to make it a tiny bit more clear to the end-user can we please change it to be `WebAuthn::AttestationStatement::FormatNotSupportedError`?